syntax = "proto3";

package util;

option java_package = "common.model.protoUtils";

message UUIDProto {
  bytes raw_uuid = 1;
}